audit-service/src/test/java/uk/gov/gchq/palisade/service/audit/service/AuditServiceTestCommon.java

```diff
@@ -114,7 +114,7 @@ static Rules mockRules() {
                 .withResourceId(mockResource().getId())
                 .withContext(mockContext())
                 .withException(mockException())
-                .withServiceClass(Service.class);
+                .withServiceName(ServiceName.USER_SERVICE.name());
     }
```

Comment:
Given this is a common utils class for testing, I think we need to make it clear that this method is registerRequestExceptionAuditRequestFromUserService. And then if we need the exception to have come from a different service then we can have another similarly named method for that.

Reply:
Done